.github/workflows/release.yml

```diff
@@ -62,7 +62,8 @@ jobs:
           aws-region: ${{ secrets.AWS_REGION }}
           role-to-assume: ${{ secrets.AWS_PUBLICECR_ROLE_ARN }}
           role-duration-seconds: 1200
-      - name: Publish adapter to public ECR
+      - if: github.ref == 'refs/heads/master'
+        name: Publish adapter to public ECR
         uses: ./.github/actions/release/publish-artifacts
         with:
           branch: ${{ fromJSON('[undefined, "develop"]')[github.ref == 'refs/heads/develop'] }}
```

Comment:
If we're going to permanently remove `develop-latest`, we could simplify the branch + latest logic so that it's
```
branch: undefined
latest: undefined
```

Reply:
Like so?